INFO     cardano_node_tests.tests.conftest:conftest.py:135 Changed CWD to '/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-15/popen-gw2'.
INFO     cardano_node_tests.utils.cluster_management:cluster_management.py:414 Stopping cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-15/cluster3/startup_files/vlqkoafx/stop-cluster-hfc`.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:340 Starting cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-15/cluster3/startup_files/vlqkoafx/start-cluster-hfc`.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:342 Cluster started.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:362 Restarting services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:177 2022-03-04 18:09:55.422791: Waiting for 200 sec for KES expiration.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:95 2022-03-04 18:13:15.537805: Waiting for the end of current epoch.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:362 Restarting services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:197 2022-03-04 18:15:57.224287: Waiting 120 secs to make sure the expected errors make it to log files.